Dockerfile

```diff
@@ -1,17 +1,21 @@
-FROM linkyard/alpine-helm:2.16.6
+FROM alpine/helm:2.14.3
```

Comment:
Is there a specific reason why you want to downgrade?

Reply:
No, but the image location should go back to alpine/helm. Version is painful before can't use new Helm clients with older server versions. Can't really win for that unfortunately and would make sense to go to the latest Helm 2 release.